guide/additional-info/async-await.md

```diff
@@ -1,6 +1,6 @@
 # Understanding async/await
 
-If you aren't very familiar with ECMAScript 2017, you may not know about async/await. It's a useful way to handle Promises in a synchronous manner instead stacking `.then()` callbacks. Also, it looks cleaner and it increases readability overall.
+If you aren't very familiar with ECMAScript 2017, you may not know about async/await. It's a useful way to handle Promises in a hoisted manner. Also, it looks cleaner, it increases readability overall and it is slightly faster.
```

Comment:
The last sounds a bit repetitive, when describing the code with "it"/"it is". I'd suggest changing it to "In addition, it looks cleaner, increases overall readability, and is slightly faster."